package.json

```diff
@@ -34,6 +34,9 @@
     "assets/**/*.scss": [
       "prettier --write",
       "git add"
+    ],
+    "controllers/routes.js": [
```

Comment:
Few thoughts on this.

1. This is in the lint-staged block, so will run on pre-commit, not pre-push. Is that intentional?
2. Should `./modules/cloudfront.js` not live in `bin/scripts` as it's executable?